src/developer/web-api/tracker.md

```diff
@@ -1345,6 +1345,12 @@ item:
         &filter=lw1SqmMlnfh:GT:150
         &filter=lw1SqmMlnfh:LT:190
 
+A query with characters in the filter value that need escaping:
```

Comment:
It helps me (and maybe other users) to see the actual value that is being sent and how the value is interpreted. I assume the value will be interpreted as `:,/`, right?

Reply:
yes

Reply:
Can you please add that the value will be interpreted as `:,/`